clients/fides-js/src/lib/consent-constants.ts

```diff
@@ -123,6 +123,12 @@ export const FIDES_OVERRIDE_OPTIONS_VALIDATOR_MAP: FidesOverrideValidatorMap[] =
       overrideKey: "fides_modal_default_view",
       validationRegex: /^\/tcf\/(purposes|features|vendors)$/,
     },
+    {
+      overrideName: "fidesModalDisplay",
+      overrideType: "string",
+      overrideKey: "fides_modal_display",
+      validationRegex: /^(immediate|banner_click)$/,
```

Comment:
sorry, my bad. Can we change "banner_click" to something else? I just realized that some experiences have no banner and it's just the Manage Preferences link on the page. "manage_preferences_click" might be too long? Or maybe just "default" for now?